pupil_src/shared_modules/player_methods.py

```diff
@@ -269,6 +271,12 @@ def update_meta_info(rec_dir, meta_info):
         csv_utils.write_key_value_file(csvfile,meta_info)
 
 
+def _update_info_version_to(new_version, rec_dir):
```

Comment:
Do you have a specific reason for the order of the arguments? Intuitively, I would have reversed the order... I cannot tell why.

Reply:
I know what you mean and I thought about this as well. I went for this order because something like _update_info_version_to('1.9', rec_dir) reads so nicely ("update info version to 1.9")

Reply:
Agreed